discord.bot/src/commands/cmdAutoChannel.java

```diff
@@ -0,0 +1,128 @@
+package commands;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import net.dv8tion.jda.core.events.message.MessageReceivedEvent;
+import objects.AutoChannel;
+import util.Static;
+
+/**
+ *
+ * @author Patrick
+ *
+ */
+public class cmdAutoChannel implements Command {
+
+	@Override
+	public boolean isCalled(String[] args, MessageReceivedEvent event) {
+		return false;
+	}
+
+	@SuppressWarnings("unchecked")
+	@Override
+	public void action(String[] args, MessageReceivedEvent event) {
+		if (args.length <= 0)
```

Comment:
klana verbesserungsvorschlag für alle `args.length` überprüfungen
```suggestion
		if (args.length == 0) {
		    event.getChannel.sendMessage(getHelp()).complete();
		    return; // durch des early return brauch ma den else block nd und i glab des liest sich besser
```